html/mycakeapp/src/Controller/UsersController.php

```diff
@@ -74,7 +74,9 @@ public function logout()
     public function beforeFilter(Event $event)
     {
         parent::beforeFilter($event);
-        $this->Auth->allow(['login', 'index', 'add']);
+        $this->Auth->allow(['login'
+        // , 'index', 'add'
+        ]);
```

Comment:
```suggestion
        $this->Auth->allow(['login', 'index', 'add'])
```
indexとaddを許可してください。レビュー時にユーザが登録できないので